src/Localisation/Languages/de.xml

```diff
@@ -0,0 +1,106 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<language id="de">
+
+  <!--Chat Panel Start-->
+  <translation id="Welcome">Willkommen beim Cities:Skylines Multiplayer!</translation>
+  <translation id="Shortcut1">Drücke die Taste ~ (Tilde), um den Chat anzuzeigen oder auszublenden.</translation>
+  <translation id="Shortcut2">Drücke die Tabulatortaste, um zum Chat zu wechseln.</translation>
+  <translation id="Discord">Tritt unserem Discord-Server bei: https://discord.gg/RjACPhd</translation>
+  <translation id="Help">Gib '/help' ein, um eine Liste der Befehle und deren Verwendung anzuzeigen.</translation>
+  <translation id="Support">Gib "/support" ein, um herauszufinden, wo du Fehler melden und Hilfe erhalten kannst.</translation>
+
+  <!--Chat Commands-->
+  <translation id="ChatCMDhelp">Zeigt Informationen darüber an, welche Befehle verwendet werden können.</translation>
+  <translation id="ChatCMDversion">Zeigt Versionsinformationen über die Mod und das Spiel an.</translation>
+  <translation id="ChatCMDsupport">Zeigt Support-Links für die Mod an.</translation>
+  <translation id="ChatCMDplayers">Zeigt eine Liste der mit dem Server verbundenen Spieler an.</translation>
+  <translation id="ChatCMDnonplayers">Sie hosten keinen Server und sind mit keinem verbunden.</translation>
+  <translation id="ChatCMDhide-chat">Versteckt das Chat-Fenster. Kann durch Drücken der Taste ~ (Tilde) wieder angezeigt werden.</translation>
+  <translation id="ChatCMDclear">Löscht alles aus dem Chat-Log.</translation>
+  <translation id="ChatCMDopen-log">Öffnet den Multiplayer-Log.</translation>
+
+  <!--Multiplayer Menu-->
+  <translation id="Multiplayer">Mehrspieler</translation>
+  <translation id="MultiplayerMenu">Mehrspieler-Menü</translation>
+  <translation id="JoinGame">Spiel beitreten</translation>
+  <translation id="HostGame">Server hosten</translation>
```

Comment:
I think `Spiel hosten` would be a better translation.

Reply:
Translation would be open to anyone finding the best context for certain text within the mod. So you're probably right.